core/services/feeds/models.go

```diff
@@ -87,11 +90,24 @@ func NewChainType(s string) (ChainType, error) {
 		return ChainTypeEVM, nil
 	case "STARKNET":
 		return ChainTypeStarknet, nil
+	case "SOLANA":
+		return ChainTypeSolana, nil
+	case "APTOS":
+		return ChainTypeAptos, nil
```

Comment:
I wonder should we add cosmos while we are here too? But obviously let try to get the PR in first so we dont need to get it approved again.